GRAPHICS/ScaleformMovieMethodAddParamPlayerNameString.md

```diff
@@ -17,3 +17,12 @@ When switching with a controller, the icons update and become the controller's i
 ## Parameters
 * **string**: 
 
+```lua
```

Comment:
Please properly format the examples as outlined [here](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md), your current example won't work.